components-core/src/main/java/com/adyen/checkout/components/base/lifecycle/ViewModelExt.kt

```diff
@@ -25,6 +25,7 @@ inline fun <reified ViewModelT : ViewModel> viewModelFactory(crossinline factory
         }
     }
 
+// TODO take from develop
```

Comment:
What do you mean?

Reply:
I had changed the extension here before rebasing it. Left a note to not forget we had done the change already in develop.